src/server/game/Server/Packets/CharacterPackets.h

```diff
@@ -53,23 +53,23 @@ namespace WorldPackets
 
                 ObjectGuid Guid;
                 std::string Name;
-                uint8 ListPosition       = 0; ///< Order of the characters in list
-                uint8 RaceID             = 0;
-                uint8 ClassID            = 0;
-                uint8 SexID              = 0;
-                uint8 SkinID             = 0;
-                uint8 FaceID             = 0;
-                uint8 HairStyle          = 0;
-                uint8 HairColor          = 0;
-                uint8 FacialHair         = 0;
-                uint8 ExperienceLevel    = 0;
-                int32 ZoneID             = 0;
-                int32 MapID              = 0;
+                uint8 ListPosition    = 0; ///< Order of the characters in list
+                uint8 RaceID          = 0;
+                uint8 ClassID         = 0;
+                uint8 SexID           = 0;
+                uint8 SkinID          = 0;
+                uint8 FaceID          = 0;
+                uint8 HairStyle       = 0;
+                uint8 HairColor       = 0;
+                uint8 FacialHair      = 0;
+                uint8 ExperienceLevel = 0;
+                int32 ZoneID          = 0;
+                int32 MapID           = 0;
                 TaggedPosition<Position::XYZ> PreloadPos;
                 ObjectGuid GuildGUID;
-                uint32 Flags             = 0; ///< Character flag @see enum CharacterFlags
-                uint32 Flags2            = 0; ///< Character customization flags @see enum CharacterCustomizeFlags
-                bool FirstLogin      = false;
+                uint32 Flags          = 0; ///< Character flag @see enum CharacterFlags
+                uint32 Flags2         = 0; ///< Character customization flags @see enum CharacterCustomizeFlags
+                bool FirstLogin       = false;
 
```

Comment:
actually you can remove these spaces entirely to follow the standards of the other packet class member files

Reply:
Order members by size, larger first (u64 > u32 > u16 > u8 > string)

Reply:
- Not entirely sure what you mean by this. Is it related to ZoneID and MapID being 32 bits in header, but stored as 16 bits in the DB? 
- I agree, however there is no alternative with the readability of TaggedPosition. (The only one would be G3D::Vector3, but TaggedPosition was implemented with the goal of removing G3D from packet headers.)
- I'll let you wrestle that out with Elon. Winner gets to decide whether there should be empty lines or not.

Reply:
He means you should order the members by type size for optimal performance. (bool, uint8, uint16, uint32 etc)
About the spacings that is for you to decide. warpten and me have some different tastes when it comes to codestyle so just pick your favourite here.

Reply:
Alright should be sorted, not sure if `PreloadPos` and `VisualItems` are located correctly.